src/rust/src/x509/sign.rs

```diff
@@ -55,15 +55,15 @@ fn identify_key_type(py: pyo3::Python<'_>, private_key: &pyo3::PyAny) -> pyo3::P
         .getattr("Ed448PrivateKey")?
         .extract()?;
 
-    if rsa_private_key.is_instance(private_key)? {
+    if private_key.is_instance(rsa_private_key)? {
```

Comment:
Flipping the `.is_instance` calls isn't mentioned in the pyo3 0.15 -> 0.16 migration guide. The new format makes way more sense (checking that the instance of the object is an instance of a type rather than calling is_instance on the type), but was it just not properly documented for  https://pyo3.rs/v0.16.2/migration.html?